ingestion/src/metadata/ingestion/source/database/postgres_lineage.py

```diff
@@ -0,0 +1,25 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Postgres lineage module
+"""
+from metadata.ingestion.source.database.lineage_source import LineageSource
+from metadata.ingestion.source.database.postgres_query_parser import (
+    PostgresQueryParserSource,
+)
+from metadata.utils.sql_queries import POSTGRES_SQL_STATEMENT
+
+
+class PostgresLineageSource(PostgresQueryParserSource, LineageSource):
+
+    sql_stmt = POSTGRES_SQL_STATEMENT
+
+    filters = ""  # No filtering in the queries
```

Comment:
@codingwithabhi here I believe we do need some filters. We only want to pick up `CREATE TABLE AS`, `INSERT INTO` and these kinds of statements.

IDK if the pg statements table has some `query_type` field we can use to filter that, otherwise we can filter the query text directly as we do in redshift for example https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/database/redshift_lineage.py#L42